chart/templates/NOTES.txt

```diff
@@ -98,7 +98,7 @@ Flower dashboard:
 {{- else }}
 You can now access your dashboard(s) by executing the following command(s) and visiting the corresponding port at localhost in your browser:
 
-Airflow Webserver:     kubectl port-forward svc/{{ include "airflow.fullname" . }}-webserver {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
+Airflow API server:     kubectl port-forward svc/{{ include "airflow.fullname" . }}-api-server {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
```

Comment:
This should have an if/else based on Airflow version since the chart supports both AF 2 and AF 3

Reply:
yes - this one indeed.

Reply:
Fixed. I also improved breeze testing so that we do not have to modfiy it every time python/k8s versions change